Add gijgo@1.3.1 w/ npm auto-update

atatanasov opened the pull request:
Library name: Gijgo
Git repository url: https://github.com/atatanasov/gijgo/
npm package name or url (if there is one): https://www.npmjs.com/package/gijgo
License (List them all if it's multiple): MIT License
Official homepage: http://gijgo.com

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:
   - Count of stars:
   - Count of forks:
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

PeterDaveHello commented:
Hi @atatanasov, I see #11076 and #11077, which one would you like to finish it? Thanks.

PeterDaveHello commented:
@maruilian11 @pvnr0082t please take a look.

PeterDaveHello commented on ajax/libs/gijgo/css/demo.css:
```diff
@@ -0,0 +1,46 @@
+﻿.gj-frm-ctrl {
```
Do we need `demo.css`? Or it's just for demo purpose?

atatanasov replied:
I'm using this css files for my demos. Not required, but nice to have.

atatanasov commented:
I created new pull request where I removed demo.css, added package.json and created better organization. Please close this one and review https://github.com/cdnjs/cdnjs/pull/11094

PeterDaveHello commented:
Move to  #11094

PeterDaveHello commented:
Move to #11110
